Packs/BitcoinAbuse/Integrations/BitcoinAbuse/BitcoinAbuse.py

```diff
@@ -264,13 +323,16 @@ def main() -> None:
     try:
 
         if command == 'test-module':
-            fetch_indicators(demisto.params(), test_module=True)
+            bitcoin_abuse_test_module_command(demisto.params())
 
         elif command == 'fetch-indicators':
-            fetch_indicators(demisto.params(), test_module=False)
+            bitcoin_abuse_fetch_indicators_command(demisto.params())
+
+        elif command == 'bitcoinabuse-get-indicators':
+            bitcoin_abuse_get_indicators_command(demisto.params(), demisto.args())
```

Comment:
```suggestion
            return_results(get_indicators_command(demisto.args()))
```

Reply:
@ShahafBenYakir what is the reason to not send demisto.params() ? it is used by the function